sw/device/tests/sensor_ctrl_alerts.c

```diff
@@ -71,6 +72,31 @@ static uint32_t get_events(dif_toggle_t fatal) {
   return events;
 }
 
+/**
+ *  Count the nubmer of 1's in a given vector.
+ *  Not attempting to be smart, just do something that is straightforward.
+ *  See https://en.wikipedia.org/wiki/Hamming_weight
+ */
+static uint32_t pop_count32(uint32_t x) {
```

Comment:
You may be able to use `bitfield_popcount32 ` from sw/device/lib/base/bitfield.h

Reply:
?! i did not know this existed. let me have a look.